plugins/outputs/d2l_kinesis/d2lKinesisOutput.go

```diff
@@ -21,6 +22,10 @@ const awsMaxRecordsPerRequest = 500
 const awsMaxRecordSize = 1048576  // 1 MiB
 const awsMaxRequestSize = 5242880 // 5 MiB
 
+type KinesisAPI interface {
```

Comment:
Use to come from `github.com/aws/aws-sdk-go/service/kinesis/kinesisiface` but they stopped doing this